tests/CMakeLists.txt

```diff
@@ -17,7 +17,7 @@ if(BUILD_TESTING)
     set_property(TARGET fixtures PROPERTY INTERFACE_LINK_LIBRARIES xdg-basedir gtest)
 
     add_executable(test_libappimage test_libappimage.cpp)
-    target_link_libraries(test_libappimage fixtures libappimage libsquashfuse)
+    target_link_libraries(test_libappimage fixtures libappimage libsquashfuse PkgConfig::glib)
```

Comment:
This dependency should be pulled in by libappimage. If it's not, you're doing something wrong, apparently. This change should not be necessary, and implies issues elsewhere.

Reply:
Agreed, fixing it